src/tally.cpp

```diff
@@ -440,14 +466,78 @@ std::string pyne::Tally::mcnp(int tally_index, std::string mcnp_version) {
     } else if (tally_type.find("Current") != std::string::npos) {
       // makes no sense in mcnp
     }
+  
+  
+  } else if (entity_type.find("Mesh") != std::string::npos) {
+    output << "FMESH4:" << particle_token << "  ";
+    output << "GEOM=";
+    std::stringstream sup_var;
+
+    if (entity_geometry.find("XYZ") != std::string::npos) {
+      output << "XYZ ";
+    } else if (entity_geometry.find("Cylinder") != std::string::npos) {
+      output << "CYL ";
+      if (axl[0] != 0 || axl[1] != 0 || axl[2] != 0) {
+        sup_var << "AXL=";
+        for (int i = 0; i < 3; i++) {
```

Comment:
Seems like it would be useful to write a method to write vectors as "a b c" triplets.